pennylane/templates/subroutines/qdrift.py

```diff
@@ -144,32 +175,28 @@ def my_circ(time):
 
     """
 
+    def _flatten(self):
+        h = self.hyperparameters["base"]
+        hashable_hyperparameters = tuple(
+            item for item in self.hyperparameters.items() if item[0] != "base"
+        )
+        return (h, self.data[-1]), hashable_hyperparameters
+
+    @classmethod
+    def _unflatten(cls, data, metadata):
+        return cls(*data, **dict(metadata))
+
     def __init__(  # pylint: disable=too-many-arguments
-        self, hamiltonian, time, n=1, seed=None, decomposition=None, id=None
+        self, hamiltonian, time, n=1, seed=None, id=None
```

Comment:
Can we add a note to the doc-string of this class to tell direct users to `qml.apply` (maybe just until the next release, then we can remove the note from the docs) since we won't have a full deprecation cycle?

Reply:
Done.